lib/index.js

```diff
@@ -352,10 +366,10 @@ internals.getExistsValues = function (type, exists) {
             return (value.isContext ? '$' : '') + value.key;
         }
 
-        return value;
+        return JSON.stringify(value);
```

Comment:
Don't you prefer doing that in the template and keep the raw object so that people providing their own templates can use that more easily ?